integrations/github/README.md

```diff
@@ -0,0 +1,21 @@
+# github-haystack
```

Comment:
I think it would be good for us to add examples here in the Readme on how to use or to link to the tutorial/google colab for how to use. 

Also another relevant detail I think is that these prompts were optimized using Anthropic models. Could be a useful thing for users to know.

Reply:
google colab in the cookbook and some more examples in an integration page is what I imagine. The README's we currently don't fill out, for example see: https://pypi.org/project/opensearch-haystack/
Might be a good idea to change that and use a copy of the integrations page. I don't see a good reason to keep it empty but I would prefer a consistent solution. I'll talk to Bilge.